btree/btree.c

```diff
@@ -6989,33 +6980,22 @@ static int64_t btree_close_tree_tick(struct m0_sm_op *smop)
 			tree_put(tree);
 			return P_DONE;
 		}
-		tree->t_starttime = m0_time_now();
 		m0_rwlock_write_unlock(&tree->t_lock);
 
 		/** put tree's root node. */
 		bnode_put(tree->t_root->n_op, tree->t_root);
 		/** Fallthrough to P_TIMECHECK */
 	case P_TIMECHECK:
```

Comment:
Since we are not checking time anymore, you might have to change the name too.

Reply:
Changed the name to WAITCHECK.